tests/ui/test_configuration_page.py

```diff
@@ -39,3 +39,19 @@ def test_openapi_json_download_button(
             download_openapi_href,
             operator="in",
         )
+
+    @pytest.mark.execute_enterprise_cloud_true
+    @pytest.mark.forwarder
+    @pytest.mark.configuration
+    def test_ucc_credits_label_exists(
+        self, ucc_smartx_selenium_helper, ucc_smartx_rest_helper
+    ):
+        """Verifies the UCC label is rendered on the page"""
+        configuration_page = ConfigurationPage(
+            ucc_smartx_selenium_helper, ucc_smartx_rest_helper
+        )
+        self.assert_util(
+            left="UCC",
+            operator="in",
+            right=configuration_page.ucc_credit.wait_to_display,
+        )
```

Comment:
We can take UCC version from `splunk_add_on_ucc_framework/__init__.py` and be a bit more precise with this test case.